src/plugin/fieldViews/__tests__/TextFieldView.spec.ts

```diff
@@ -313,6 +313,31 @@ describe("the TextFieldView, as an extension of the ProseMirrorFieldView", () =>
     expect(removedMarks).toStrictEqual(null);
   });
 
+  it("should update the storedMarks in the outer editor state when the inner editor sets a storedMark", () => {
+    const {
+      view,
+      node,
+      decorations,
+      fieldView,
+      offset,
+      textFieldViewInnerEditor,
+    } = getEditorWithTextField();
+    const initialInnerMarks = textFieldViewInnerEditor.state.storedMarks;
+    const initialOuterMarks = view.state.storedMarks;
+
+    const exampleMark = testSchema.marks.strike.create();
+    fieldView.onUpdate(node, offset, decorations, undefined, [exampleMark]);
+    const markTr = textFieldViewInnerEditor.state.tr.addStoredMark(exampleMark);
+    textFieldViewInnerEditor.dispatch(markTr);
+    const updatedInnerMarks = textFieldViewInnerEditor.state.storedMarks;
+    const updatedOuterMarks = view.state.storedMarks;
+
+    expect(initialInnerMarks).toBe(null);
+    expect(initialOuterMarks).toBe(null);
+    expect(updatedInnerMarks).toStrictEqual([exampleMark]);
+    expect(updatedOuterMarks).toStrictEqual([exampleMark]);
+  });
```

Comment:
```suggestion
  it("should update the storedMarks in the outer editor state when the inner editor sets a storedMark", () => {
    const {
      view,
      textFieldViewInnerEditor,
    } = getEditorWithTextField();
    const initialInnerMarks = textFieldViewInnerEditor.state.storedMarks;
    const initialOuterMarks = view.state.storedMarks;

    expect(initialInnerMarks).toBe(null);
    expect(initialOuterMarks).toBe(null);

    const exampleMark = testSchema.marks.strike.create();
    const markTr = textFieldViewInnerEditor.state.tr.addStoredMark(exampleMark);
    textFieldViewInnerEditor.dispatch(markTr);
    const updatedInnerMarks = textFieldViewInnerEditor.state.storedMarks;
    const updatedOuterMarks = view.state.storedMarks;

    expect(updatedInnerMarks).toStrictEqual([exampleMark]);
    expect(updatedOuterMarks).toStrictEqual([exampleMark]);
  });
```

This test looks great! An optional suggestion to –
- remove unused destructured params
- remove L329, which I think is not needed (running locally the test passes without it – this matches my understanding that setting the stored mark should be sufficient)
- move the initial assertions closer to their declarations for readability